data/individual-routes/liquid-pipelines/P6505.geojson

```diff
@@ -0,0 +1,352 @@
+{
+    "type": "FeatureCollection",
+    "generator": "JOSM",
+    "features": [
+        {
+            "type": "Feature",
+            "properties": {
+                "layer": "-1",
+                "location": "underground",
+                "man_made": "pipeline",
+                "name": "Leveret Pipeline"
+            },
+            "geometry": {
+                "type": "LineString",
+                "coordinates": [
+                    [
+                        -103.90634180188,
+                        31.80982207214
+                    ],
+                    [
```

Comment:
```suggestion
                "coordinates": [
                    [
```

You can make inline suggestions when reviewing diffs. Click and drag to select lines of code, and hit Ctrl+G. Here's an example (please don't merge) that deletes the first coordinate from this LineString.

Reply:
Got it, thanks for pointing that out!